### ISSUE TITLE
Allow terminal link detection commands to read links from more than just the viewport

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#632]
Problem reported via https://github-com/microsoft/vscode/-/149882-issuecomment-1137054006

This would be helpful especially for screen reader users.

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold

**Comment by vscodenpa** [comment]
confidence for label accessibility: 0.6. does not meet threshold

**Comment by vscodenpa** [comment]
confidence for assignee Tyriar: 0.85. does meet threshold
